Makefile.am

```diff
@@ -10,6 +10,7 @@ pkgconfig_DATA = libfastjson.pc
 
 libfastjsonincludedir = $(includedir)/libfastjson
 libfastjsoninclude_HEADERS = \
+	atomic.h \
 	arraylist.h \
```

Comment:
I think we should avoid using it here, I have already worked on the ability to not provide json_object_private.h. Especially this header, I think, should not be visible to callers - it prevents changing many implementation details. I'll investigate further today.